src/main/resources/templates/start.html

```diff
@@ -34,7 +34,10 @@
   <div class="input-group-prepend">
     <span class="input-group-text" id="inputGroup-sizing-sm">Gender</span>
   </div>
-  <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name = "gender" th:field="*{gender}">
+  <select th:field="*{gender}">

```

Comment:
For a limited set of choices, a drop down is always better than free-hand text.  Less likely to make an input mistake that way.